interface/forms/eye_mag/view.php

```diff
@@ -162,134 +167,25 @@
-    <script src="<?php echo $GLOBALS['assets_static_relative'] ?>/qtip2-2-2-1/jquery.qtip.min.js"></script>
-    <script type="text/javascript" src="../../../library/dialog.js?v=<?php echo $v_js_includes; ?>"></script>
 
-    <script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative'] ?>/jscolor-2-0-4/jscolor.js"></script>
 
-    <script language="JavaScript">
-    <?php require_once("$srcdir/restoreSession.php");
-    ?>
-    function dopclick(id) {
-        <?php if ($thisauth != 'write') : ?>
-      dlgopen('../../patient_file/summary/a_issue.php?issue=0&thistype=' + id, '_blank', 550, 400);
-        <?php else : ?>
-      alert("<?php echo xls('You are not authorized to add/edit issues'); ?>");
-        <?php endif; ?>
-    }
-    function doscript(type,id,encounter,rx_number) {
-      dlgopen('../../forms/eye_mag/SpectacleRx.php?REFTYPE=' + type + '&id='+id+'&encounter='+ encounter+'&form_id=<?php echo attr(addslashes($form_id)); ?>&rx_number='+rx_number, '_blank', 660, 590);
-    }
-
-    function dispensed(pid) {
-      dlgopen('../../forms/eye_mag/SpectacleRx.php?dispensed=1&pid='+pid, '_blank', 560, 590);
-    }
-     // This invokes the find-code popup.
-    function sel_diagnosis(target,term) {
-      if (target =='') {
-          target = "0";
-      }
-      IMP_target = target;
-        <?php
-        if ($irow['type'] == 'PMH') { //or POH
-        ?>
-          dlgopen('<?php echo $rootdir ?>/patient_file/encounter/find_code_popup.php?codetype=<?php echo attr(collect_codetypes("medical_problem", "csv")) ?>&search_term='+encodeURI(term), '_blank', 600, 400);
-            <?php
-        } else {
-            ?>
-          dlgopen('<?php echo $rootdir ?>/patient_file/encounter/find_code_popup.php?codetype=<?php echo attr(collect_codetypes("diagnosis", "csv")) ?>&search_term='+encodeURI(term), '_blank', 600, 400);
-            <?php
-        }
-        ?>
-    }
-
-    var obj =[];
-    <?php
-      //also add in any obj.Clinical data if the form was already opened
-      $codes_found = start_your_engines($encounter_data);
-    if ($codes_found) { ?>
-      obj.Clinical = [<?php echo json_encode($codes_found[0]); ?>];
-    <?php
-    } ?>
-
-    </script>
+      <title> <?php echo xlt('Chart'); ?>: <?php echo text($pat_data['fname'])." ".text($pat_data['lname'])." ".text($visit_date); ?></title>
+      <link rel="shortcut icon" href="<?php echo $GLOBALS['images_static_relative']; ?>/favicon.ico" />
+      <meta charset="utf-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="description" content="OpenEMR: Eye Exam">
+      <meta name="author" content="OpenEMR: Ophthalmology">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <!-- Add Font stuff for the look and feel.  -->
+      <?php Header::setupHeader([ 'jquery-ui', 'jquery-ui-redmond','datetime-picker', 'dialog' ,'jscolor', 'qtip2' ]); ?>
```

Comment:
woohooo. nice to see this.
Since you are using another jquery version below, place the following token in above setupHeader(so will avoid clashes and also will flag is in future to upgrade this script to more modern jquery):
no_jquery